OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeploy.cs

```diff
@@ -63,6 +63,12 @@ public class GrantConditionOnDeployInfo : PausableConditionalTraitInfo, IEditorA
 		[Desc("Undeploy before the actor is picked up by a Carryall?")]
 		public readonly bool UndeployOnPickup = false;
 
+		[Desc("Deploy when the actor attacks?")]
+		public readonly bool DeployOnAttack = false;
+
+		[Desc("On which armament tigger deploying?")]
```

Comment:
```suggestion
		[Desc("On which armament trigger deploying?")]
```

I am also not a native speaker, but it sounds a bit broken.